centralized-sampling-tests/sample-apps/spring-boot/src/main/java/com/amazon/sampleapp/AppController.java

```diff
@@ -0,0 +1,132 @@
+package com.amazon.sampleapp;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.contrib.awsxray.AwsXrayRemoteSampler;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.resources.Resource;
+import io.opentelemetry.sdk.trace.SdkTracerProvider;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import java.time.Duration;
+
+// Controller for the application
+@Controller
+public class AppController {
+  // Resource used to create xray sampler
+  private Resource resource;
+
+  // Opentelemetry builder to create a xray remote sampler with polling interval of 1
+  private OpenTelemetry openTelemetry;
```

Comment:
Can we remove private attributes resource and opentelemetry, just autowire a Tracer?